README.md

```diff
@@ -1 +1,41 @@
-A MongoDB Dialect for the Hibernate ORM
+# A MongoDB Dialect for the Hibernate ORM
+
+This project aims to provide a library to seamlessly integrate MongoDB with Hibernate ORM. Hibernate _ORM_ is a powerful **O**bject-**r**elational **m**apping tool. Due to the SQL and JDBC standards, Hibernate ORM could centralize each SQL vendor's idiosyncrasies in the so-called _Hibernate Dialect_. But this project will include a NoSQL member in the Hibernate's Dialect family.
+
+## Overview
+
+MongoDB speaks _MQL_ (**M**ongoDB **Q**uery **L**anguage in JSON format) instead of SQL, and lacks a full-fledged JDBC Driver currently. However, MongoDB does support db transaction and table joins simulation. This project will create a NoSQL Hibernate Dialect by bridging the gaps, particularly:
```

Comment:
I find the first two sentences more confusing than enlightening.  

* Starting by saying that we lack a full-fledged JDBC driver inspires more fear than confidence, so let's not mention it.
* db transactions and table join simulation are just two of the many, many, many features that MQL/MongoDB provide that make Hibernate a good fit.

Not sure how to fix it, but can you try again?